package.json

```diff
@@ -3,7 +3,7 @@
   "devDependencies": {
     "husky": "^2.3.0",
     "lint-staged": "^8.1.7",
-    "prettier": "1.17.1"
+    "prettier": "^2.0.1"
```

Comment:
Please, revert this change and do as we discussed in chat:

1. We merge all (most) the existing PRs
2. Create a separate PR where only prettier changes will occur

This PR is impossible to review at this state.